nvflare/private/fed/server/server_engine.py

```diff
@@ -224,13 +240,13 @@ def _listen_command(self, listen_port, run_number):
             job_id = self.run_processes.get(run_number).get(RunProcessKey.JOB_ID)
             try:
                 if conn.poll(0.1):
-                    received_data = conn.recv()
+                    received_data = self._recv(conn, job_id, run_number)
                     command = received_data.get(ServerCommandKey.COMMAND)
                     data = received_data.get(ServerCommandKey.DATA)
 
                     if command == ServerCommandNames.GET_CLIENTS:
                         return_data = {ServerCommandKey.CLIENTS: clients, ServerCommandKey.JOB_ID: job_id}
-                        conn.send(return_data)
+                        self._send(conn, job_id, run_number, return_data)
```

Comment:
send signature is `def _send(self, conn, data, job_id, run_number):`

Reply:
woops. let me change